cmd/gossamer/configcmd.go

```diff
@@ -69,12 +70,28 @@ func makeNode(ctx *cli.Context, gen *genesis.GenesisState) (*dot.Dot, *cfg.Confi
 	// set up message channel for p2p -> core.Service
 	msgChan := make(chan []byte)
 
-	// TODO: trie and runtime
+	if gen == nil {
+		return nil, nil, errors.New("genesis is nil")
+	}
+
+	if gen.GenesisTrie == nil {
+		return nil, nil, errors.New("no genesis trie exists")
+	}
+
+	// load runtime code from trie and create runtime executor
+	code, err := gen.GenesisTrie.Get([]byte(":code"))
+	if err != nil {
+		log.Crit("error retrieving :code from trie", "error", err)
```

Comment:
We should avoid logging here and rather return errors, this ensures we will get a single error log and then it will exit

Reply:
done